WebApplication1/Controllers/BooksController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using WebApplication1.Services;
+using WebApplication1.BookData;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace WebApplication1.Controllers
+{
+    [Route("api/[controller]")]
+    public class BooksController : Controller
+    {
+        BookServices Services = new BookServices();

```

Comment:
Services may be changed to service as service can have many functionalities

Reply:
It has many functionalities, and hence the name.